rest/model/createhost.go

```diff
@@ -6,16 +6,14 @@ import (
 )
 
 type CreateHost struct {
-	DNSName    string `json:"dns_name"`
-	IP         string `json:"ip_address"`
-	InstanceID string `json:"instance_id"`
+	DNSName    string `json:"dns_name,omitempty"`
+	IP         string `json:"ip_address,omitempty"`
+	InstanceID string `json:"instance_id,omitempty"`
 
-	HostID      string `json:"host_id"`
-	ContainerID string `json:"container_id"`
-	ParentID    string `json:"parent_id"`
-	Image       string `json:"image"`
-	Command     string `json:"command"`
-	Port        int    `json:"port"`
+	HostID   string `json:"host_id,omitempty"`
+	ParentID string `json:"parent_id,omitempty"`
+	Image    string `json:"image,omitempty"`
+	Command  string `json:"command,omitempty"`
```

Comment:
we've historically avoided using omit-empty on rest route models and have instead used APIString, which operates as a nullable string type. 

Is it possible to not use omitempty?

Reply:
So if these fields were logged, would they show up even if they were empty strings? Only a few of these fields are usually populated so I don't want a bunch of empty fields cluttering up task logs.